samples/react-native/src/App.tsx

```diff
@@ -318,7 +318,6 @@ function App() {
   return (
     <>
       <RootNavigationContainer />
-      <RunningIndicator />
```

Comment:
Hey @krystofwoldrich 👋 

though this is still draft I'd like your feedback since I'm facing an issue with the sample app when the `RunningIndicator` is rendered. I guess this might be too specific to our sample but I was wondering if you have any ideas on how to overcome this since similar issues might occur for other apps too.

When the running indicator is removed (as in the current state of this PR) the form is showing/hiding as expected. When the indicator is added back the `showFeedbackForm` does nothing 🤔

Reply:
I've noticed there is open issue in the Reanimated repository, with a similar issue.

The recommendation is to wrap the modal in a view. 

I've tried that and it works for our case as well.

https://github.com/software-mansion/react-native-reanimated/issues/6035#issuecomment-2579469189

Reply:
Thank you for all your help on this Krystof 🙇 
Wrapping the modal in a `View` [as suggested](https://github.com/getsentry/sentry-react-native/pull/4483#discussion_r1929211735) worked perfectly :)